package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "",
+  "version": "",
+  "description": "",
+  "author": "",
+  "license": "",
+  "engines": {
+    "node": ">=6.13.0"
+  },
+  "scripts": {
+    "start": "nodemon server/index.js",
+    "dev-react": "webpack -d --watch",
+    "db-setup": "node database/seed.js"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@babel/core": "^7.7.4",
+    "@babel/preset-env": "^7.7.4",
+    "@babel/preset-react": "^7.7.4",
+    "babel-loader": "^8.0.6",
+    "body-parser": "^1.19.0",
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
+    "express": "^4.17.1",
+    "jquery": "^3.4.1",
+    "mysql2": "^2.0.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "request": "^2.88.0",
+    "sequelize": "^5.21.2",
+    "webpack": "^4.41.2",
+    "webpack-cli": "^3.3.10"
+  }
```

Comment:
What was your reasoning behind saving everything as dev-dependencies? I think other than babel/webpack/babel-loader should be in dependencies but it might not really matter.